packages/datatrak-web/src/api/mutations/useRegister.ts

```diff
@@ -15,7 +15,7 @@ type RegisterUserBody = {
 export const useRegister = () => {
   return useMutation<any, Error, RegisterUserBody, unknown>(
     (data: RegisterUserBody) => {
-      return post('signup', {
+      return post('user', {
```

Comment:
Also fixing in datatrak, pretty sure it has the same problem.

Removing this mean it will forward to central by default which is what I want